### PR TITLE
RELATED: RAIL-551 Handle table with attributes only

### PR DESCRIPTION
**Pull request by vaclavbohac**

### CODE REVIEW

**Comment by vaclavbohac** [comment]
`3.0.0-vaclavbohac-handle-empty-table-2017-12-11T16-03-57-083Z`

File: src/helpers/tests/errorHandlers.spec.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 import { Execution } from '@gooddata/typings';
 import { ErrorCodes as DataErrorCodes } from '@gooddata/data-layer';
-import { emptyResponse } from '../../execution/fixtures/ExecuteAfm.fixtures';
+import {
+    emptyResponse,
+    emptyResponseWithNull,
+    attributeOnlyResponse
+} from '../../execution/fixtures/ExecuteAfm.fixtures';
 import { ErrorCodes, ErrorStates } from '../../constants/errorStates';
+import {} from 'jest';
```
Comment (vojty) [inline src/helpers/tests/errorHandlers.spec.ts]: i dont like this

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/180/ : SUCCESS in 26s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/814/ : SUCCESS in 6m 50s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/129/ : SUCCESS in 2m 15s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/184/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-el7/817/ : SUCCESS in 1m 43s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/132/ : ABORTED in 2m 51s (non-voting)

**Pull request merged**